shell/browser/browser_linux.cc

```diff
@@ -25,11 +29,19 @@
 
 namespace electron {
 
+const char kFreedesktopPortalName[] = "org.freedesktop.portal.Desktop";
+const char kFreedesktopPortalPath[] = "/org/freedesktop/portal/desktop";
+const char kFreedesktopPortalBackground[] = "org.freedesktop.portal.Background";
```

Comment:
Since these only supposed to be visible to this file, move 'em into the  anonymous namespace.